doc/html/glossary.html

```diff
@@ -848,18 +848,19 @@ <H1 ALIGN=Center>
       <TD BGCOLOR=yellow><A NAME="P">P</A></TD>
       <TD><DL>
 	  <DT>
-	    <A NAME="Panose">Panose</A>
-	  <DD>
-	    A system for describing fonts. See HP's <A HREF="http://www.panose.com/">PANOSE
-	    classification metrics guide</A>, AGFA's
-	    <A HREF="http://www.agfamonotype.com/hardware/pan1.asp">PANOSE classification
-	    metrics guide </A>and MicroSoft's
-	    <A HREF="http://msdn.microsoft.com/library/default.asp?url=/library/en-us/gdi/fontext_48aa.asp">PANOSE
-	    classification in Windows</A>. There is also an extension called
-	    <A HREF="http://www.w3.org/Fonts/Panose/pan2.html">Panose 2</A>.
+	    <A NAME="Panose">PANOSE</A>
+	  <DD>
+	    A system for describing fonts. See HP's <A HREF="http://panose.com/">PANOSE
+	    Classification Metrics Guide</A>, MicroSoft's
```

Comment:
Microsoft dropped the MicroSoft spelling a few years back. It's now just Microsoft.